src/components/Header/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { useTranslation } from 'react-i18next';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHome } from '@fortawesome/free-solid-svg-icons';
+import { Link, useLocation } from 'react-router-dom';
+
+import { useSelector } from 'contexts/UserContext';
+import PATHS from 'components/Routes/paths';
+
+import itemsMenu from './constants';
+import styles from './styles.module.scss';
+
+const classLink = (actualPath: string, path: string) =>
+  actualPath === path ? `${styles.link} ${styles.active}` : styles.link;
+
+function Header() {
+  const { t } = useTranslation();
+  const user = useSelector((state) => state.user);
+
+  const items = itemsMenu.filter(({ isProtected }) => (user && isProtected) || (!user && !isProtected));
+  const { pathname } = useLocation();
+
+  return (
+    <div className={styles.contentHeader}>
+      <div className={styles.logo}>conduit</div>
+      <div>
+        <ul className={styles.listItems}>
+          <li className={styles.item}>
+            <Link to={PATHS.home} className={classLink(pathname, PATHS.home)}>
```

Comment:
Here you can use NavLink to have the class active
https://reactrouter.com/web/api/NavLink

Reply:
Very Good!!